lib/parachain/types.go

```diff
@@ -0,0 +1,392 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package parachain
+
+import (
+	"fmt"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/crypto/sr25519"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+)
+
+// TODO: resolve these
+// https://github.com/ChainSafe/gossamer/pull/3241#discussion_r1187883189
+// https://github.com/ChainSafe/gossamer/pull/3297#discussion_r1214740051
```

Comment:
remove?

Reply:
Keeping this in for reference https://github.com/ChainSafe/gossamer/pull/3297#discussion_r1214740051